.github/workflows/website-new-version.yml

```diff
@@ -0,0 +1,54 @@
+name: Build & Deploy spec to Github Pages
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+    branches:
+      - docs/versionable-docs # testing phase only
+
+permissions:
+  id-token: write
+
+jobs:
+  site-deploy:
+    name: Build and Publish To openlineage-site repo
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          path: 'source'
+          fetch-depth: 0
+          fetch-tags: true
+      - uses: actions/checkout@v4
+        with:
+          repository: 'pawel-big-lebowski/openlineage-site'
+          path: 'target'
+          token:  ${{ secrets.ACTIONS_GITHUB_TOKEN }}
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
+          cache: "npm"
+          cache-dependency-path: source/website/package-lock.json
+      - name: Install dependencies
+        run:  |
+          cd target
+          yarn install --frozen-lockfile
+      - name: Generate new version
+        run: |
+          cd source
+          LATEST_VERSION=$(git tag --sort=taggerdate | tail -1)
```

Comment:
`${GITHUB_REF#refs/*/}"` maybe?

Reply:
this solves the problem of creating initial version. In this case we want to release last tag. Anyway, I think it's easier to have script idempotent.